vault/barrier_aes_gcm.go

```diff
@@ -86,12 +89,17 @@ func NewAESGCMBarrier(physical physical.Backend) (*AESGCMBarrier, error) {
 // Initialized checks if the barrier has been initialized
 // and has a master key set.
 func (b *AESGCMBarrier) Initialized(ctx context.Context) (bool, error) {
+	if b.initialized.Load() {
```

Comment:
I can see why using Uber's atomic package is convenient here since we want to return bool, but I wonder if we could simply use the stdlib's `atomic.LoadUint32` vs importing an additional library.

Reply:
It was Jeff's suggestion.  He likes the API the library provides and thought this would be a convenient occasion to give it a try.  I concurred.